spec/requests/api/v1/keywords_controller_spec.rb

```diff
@@ -0,0 +1,94 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe API::V1::KeywordsController, type: :request do
+  describe '#index' do
```

Comment:
For API we, usually add the request type also
```suggestion
  describe 'GET #index' do
```

Reply:
Changed :)